spec/controllers/identifications_controller_api_spec.rb

```diff
@@ -196,15 +196,6 @@
   end
 
   describe "destroy" do
-    before(:all) do
-      # some identification deletion callbacks need to happen after the transaction is complete
-      DatabaseCleaner.strategy = :truncation
-    end
-
-    after(:all) do
-      DatabaseCleaner.strategy = :transaction
-    end
-
```

Comment:
So does `test_after_commit` run everything in a complete transaction instead of using `ROLLBACK`? Is that going to slow down the test suite?

Reply:
I don't think that using the gem is equivalent to running with the `truncation` strategy all the time, but I'm not also completely sure how the gem works. Specs are quite a bit slower with the gem. I was considering doing something different and using the `truncation` strategy any time has_subscribers was enabled, but that seemed like adding yet more special cases for spec. It sounds like in Rails 5 and above they do something internally to make sue after_commits run in specs, like the gem accomplishes. I also like that when I added the gem I found we had one spec that was passing but should be failing.

If we don't mind the gem, one thing we could do to offset the increase testing time is split up specs into a couple more test groups

Reply:
Simpler tests + exposing bugs - slower tests > 0, I guess.